handbook/benefits.md

```diff
@@ -82,21 +150,21 @@ In alignment with our core value, **“We Work Hard and Go Home”**, dbt Labs o
 - At the end of each quarter, the People team will be sending managers a list of how much time off each person has already taken / planned to take for the rest of the year (based on what is requested in ADP).
 - We encourage managers to do check-ins with their team (based on the data provided above): for those not taking time off, managers should seek to understand why this is the case and to help their team member structure their workload or work priorities in a way that allows them to actually disconnect from work to recharge.
 
-We ask that you input your time-off requests into ADP and categorize them accordingly [based on the definitions below](#Types-of-time-off-plans-&-definitions). 
+We ask that you input your time-off requests into ADP and categorize them accordingly [based on the definitions below](#%EF%B8%8Ftypes-of-time-off-plans--definitions). 
 
 **Please discuss your time-off schedule with your manager prior to taking the time off so we can plan for coverage while you are out**. **Manager approval will be required to take time off!** When planning your time off, we ask that you also be mindful of the major dates and deadlines that may be unique and critical to your role.
 
 When in doubt, please discuss with your manager about your plans
 
-### ❓When would time off be declined?
+#### When would time off be declined?
 
 - It shows blatant abuse of the unlimited time off policy. For example (but not limited to the following):
     - Using open vacation in place of a leave of absence (see below).
     - Requesting time off that is more than 4 consecutive weeks in duration.
 - Amount of (or dates of request for) PTO negatively impacts your team or the company.
```

Comment:
Change to: 

- Amount of (or dates of request for) PTO negatively impacts your team or the company, **as determined by your manager.**